README.md

```diff
@@ -82,15 +82,15 @@ plugins=(
 )
 ```
 
-_Note that the plugins are separated by whitespace. **Do not** use commas between them._
+_Note that the plugins are separated by whitespace (spacebar/enterkey). **Do not** use commas between them or it will break._
```

Comment:
```suggestion
_Note that the plugins are separated by whitespace (spaces, tabs, new lines...). **Do not** use commas between them or it will break._
```